packages/neutrino/bin/start.js

```diff
@@ -33,6 +33,13 @@ module.exports = (middleware, args) => {
 
       if (!compiler.options.devServer) {
         spinner.succeed('Build completed');
+        const building = ora('Source changed, re-compiling');
```

Comment:
Should this say "Waiting for initial build to finish" or similar, instead?

Reply:
Or "Performing initial build" etc

Reply:
Oops, good catch.